CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
 ### Changed
 
 * Support for TLS-SNI-01 has been removed from all official Certbot plugins.
+* CLI flags `--tls-sni-01-port` and `--tls-sni-01-address` are now no-op, and will
+  generate a deprecation warning if used (planed to be removed with 0.34.0).
```

Comment:
I personally think removing them in the next release is too fast. I have the same opinion as I did on the `acme` code where I think 3 - 6 months makes sense and I almost prefer not including an exact version when it will be removed to give us flexibility.

Reply:
Done.